TicTacToeProject/Board.cs

```diff
@@ -17,32 +17,53 @@ public Board()
             }
         }
 
-
-        public bool isEmpty()
+        public bool isBoardEmpty()
```

Comment:
For what it's worth, and no need to go back and change it, but I personally prefer what you had before `isEmpty` and `isFilled`. The "Board" part is superfluous because I'm calling it on a board object.